src/lava/magma/core/process/process.py

```diff
@@ -444,6 +458,23 @@ def is_compiled(self):
         """Returns True if process has been compiled."""
         return self._is_compiled
 
+    @property
+    def folded_view(self) -> ty.Type["AbstractProcess"]:
+        """ Return folded view process"""
+        return self._folded_view
+
+    @folded_view.setter
+    def folded_view(self, folded_view : ty.Optional[AbstractProcess] = None):
```

Comment:
Do we intend to have people calling `process.folded_view()` without a value to set folded view to `None`?

`self._folded_view` is initialized to `None` on definition on line 215: `self._folded_view : ty.Optional[AbstractProcess] = None`

I'd assume we always want a value when calling a setter to a property instead of defaulting to None that was already previously defined:
`folded_view : ty.Optional[AbstractProcess] = None`

Reply:
Yes.  The consumers update the folded_view related values through both getter and setter.